syntax = "proto3";

package helloworld;

service Greeter{
  rpc SayHi(HelloReq) returns (HellResp) {}
}

message HelloReq{
  string name = 1;
}

message HellResp{
  string message = 1;
}